src/main/kotlin/io/github/rybalkinsd/kohttp/ext/StringExt.kt

```diff
@@ -61,8 +63,10 @@ fun String.httpGet(client: Call.Factory = defaultHttpClient): Response =
  * @since 0.1.0
  * @author sergey
  */
-fun String.asyncHttpGet(client: Call.Factory = defaultHttpClient): Deferred<Response> =
-    GlobalScope.async(context = Unconfined) {
-        client.suspendCall(Request.Builder().url(this@asyncHttpGet).build())
-    }
+fun String.httpGetAsync(
+        client: Call.Factory = defaultHttpClient
```

Comment:
what is the reason of reformatting code here?

Reply:
Same as before - https://github.com/rybalkinsd/kohttp/pull/99#discussion_r289618474